### DISCUSSION TITLE
[ANSWERED] Do Brownie and other tools work correctly on M1 Macs?

### DISCUSSION THREAD

**Discussion by ZeroEkkusu** [smartcontractkit/full-blockchain-solidity-course-py#513] [Q&A] [answered]
Gm, frens, has anyone followed the course on an M1 Mac machine? I'm on a PC and want to make the switch (for the first time). I've read there might be some issues with certain libraries, etc. What is your first hand experience? Thank you.

**Comment by cromewar** [comment]
Hello @ZeroEkkusu I've heard that M1 users may have problems with libraries like py-solcx, but libraries like brownie and truffle should work fine.

**Comment by ZeroEkkusu** [comment, marked as answer]
Hey, @owersbrett, I saw you're on an M1 Mac! Are there no _critical_ issues?

**Reply by owersbrett** [reply]
Hey! Sorry for the delay. I've found no critical issues! Every time I run into something a small amount of Googling usually suffices. If you run into something, feel free to message me as I may have run into it on my way! I'm currently on hour 15 of this course

**Reply by ZeroEkkusu** [reply]
Thank you very much, Brett

**Comment by sevamove** [comment]
Hello @ZeroEkkusu, I can not say a lot about development on M1, but I tried to open an issue #287 on Raspberry Pi problem wherein I also mentioned M1 whether anybody gets this `SolcInstallationError` or not.

**Reply by ZeroEkkusu** [reply]
I read it a few days ago
